python/packages/sdk/serverless_sdk/lib/error_captured_event.py

```diff
@@ -51,19 +53,20 @@ def create(
     from .. import serverlessSdk
 
     if (
-        origin == "nodeConsole"
+        origin == "pythonConsole"
```

Comment:
In Node.js this marks logs coming from captured `console.*` (native Node.js interface) calls. 

At this moment it's not applicable in Python, although one of the next steps will be to instrument default login interface in Python (I will open an issue for that)